vendor/engines/projects/spec/controllers/projects/projects_controller_spec.rb

```diff
@@ -0,0 +1,115 @@
+require "rails_helper"
+require_relative "../../projects_spec_helper"
+
+RSpec.describe Projects::ProjectsController, type: :controller do
```

Comment:
I like having the spec structure reflect the files they relate to. Honestly the thing I like most is that using this convention, https://github.com/tpope/vim-rails picks up on the two files being related.